doc/articles/guides/06-logging.md

```diff
@@ -0,0 +1,188 @@
+# Logging
```

Comment:
Great much-needed doc. 👍  I think this should be a standalone doc, rather than part of the Silverlight tutorial. Can you move it to the parent 'articles' folder and rename it to `logging.md'? (And obviously link to it from the Silverlight tutorial - you probably have a better idea where that could/should go)